c2corg_ui/static/partials/cards/outings.html

```diff
@@ -10,10 +10,13 @@
     <span class="list-item-lang" ng-if="cardCtrl.lang != locale.lang">({{::locale.lang}})</span>
     <br>
     <div class="quality" ng-if="::doc['quality']" ng-class="::doc['quality']"
-         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left"></div>
+         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left">
+    </div>
 
-    <span class="elevation-max" ng-if="::doc['elevation_max']" uib-tooltip="{{'elevation_max' | translate}}">
-      <span translate class="value-title glyphicon glyphicon-sort-by-attributes"></span>
+    <!--elevation-max is shown only in the home feed-->
+    <span class="elevation-max" ng-if="::doc['elevation_max'] && cardCtrl.currentLocation == '/'"
```

Comment:
> In this case it seems to be robust enough - elevation-max is shown only in the home feed, and the only page where window.location.pathname == / is the home feed page.

What about the profile feed? It should work the same way than the home page feed.
For me testing the URL is not a robust way.

But why do we need to have the elevation max only in the feed?